cwms-radar-client/src/test/resources/radar/v1/json/virtual_outlets.json

```diff
@@ -0,0 +1,134 @@
+[
+  {
+    "project-id": {
+      "office-id": "SPK",
+      "name": "BIGH"
+    },
+    "virtual-outlet-id": {
+      "office-id": "SPK",
+      "name": "Compound Tainter Gates"
+    },
+    "virtual-records": [
+      {
+        "outlet-id": {
+          "office-id": "SPK",
+          "name": "TG1"
+        },
+        "downstream-outlet-ids": [
+          {
+            "office-id": "SPK",
+            "name": "TG2"
+          }
+        ]
+      },
+      {
+        "outlet-id": {
+          "office-id": "SPK",
+          "name": "TG2"
+        },
+        "downstream-outlet-ids": [
+          {
+            "office-id": "SPK",
+            "name": "TG3"
+          }
+        ]
+      },
+      {
+        "outlet-id": {
+          "office-id": "SPK",
+          "name": "TG3"
+        },
+        "downstream-outlet-ids": [
+          {
+            "office-id": "SPK",
+            "name": "TG4"
+          },
+          {
+            "office-id": "SPK",
+            "name": "TG5"
+          }
+        ]
+      },
+      {
+        "outlet-id": {
+          "office-id": "SPK",
+          "name": "TG4"
+        },
+        "downstream-outlet-ids": []
+      },
+      {
+        "outlet-id": {
+          "office-id": "SPK",
+          "name": "TG5"
+        },
+        "downstream-outlet-ids": []
+      }
+    ]
+  },
+  {
+    "project-id": {
+      "office-id": "SPK",
+      "name": "BIGH-TEST12"
```

Comment:
This is still the project id, so update line 75 instead of this one.  I'd also recommend tweaking the records a bit, so you have two unique virtual outlets.

An option for the name would be Compound Conduit Gates and the acronyms could be CG[X].

Reply:
Talk to me about how to modify the records

Reply:
Changed naming to CG[X] and renamed value as suggested